x-pack/filebeat/module/cisco/amp/test/cisco_amp2.ndjson.log-expected.json

```diff
@@ -0,0 +1,5676 @@
+[
+    {
+        "@timestamp": "2021-01-15T11:59:52.000Z",
+        "cisco.amp.computer.active": true,
+        "cisco.amp.computer.connector_guid": "test_connector_guid",
+        "cisco.amp.computer.external_ip": "8.8.8.8",
+        "cisco.amp.computer.network_addresses": [
+            {
+                "ip": "10.10.10.10",
+                "mac": "87:c2:d9:a2:8c:74"
+            }
+        ],
+        "cisco.amp.connector_guid": "test_connector_guid",
+        "cisco.amp.event_type_id": 1107296344,
+        "cisco.amp.related.mac": [
+            "87:c2:d9:a2:8c:74"
+        ],
+        "cisco.amp.tactics": [
```

Comment:
are any of the `threat` fields usable? Looks like they might not be unless ECS moves towards making the whole field set flattened/nested?

Reply:
Yeah unfortunately leaving them there was a conscious choice, the current format only fits a flattened datatype. I will bring it up with the ECS team however, to see what we can do for the future.